src/eureka/S3_data_reduction/niriss.py

```diff
@@ -1,8 +1,8 @@
 # NIRISS specific rountines go here
 #
 # Written by: Adina Feinstein
-# Last updated by: Adina Feinstein
-# Last updated date: February 7, 2022
+# Last updated by: Taylor Bell
+# Last updated date: Sept 09, 2024
```

Comment:
I agree.

On a similar note, I've also been inclined to remove all of the `History` lines from the function/class docstrings since we hardly ever bother to update them and GitHub does all the tracking for us anyway. We don't need to do so right now, but I'm game to do so whenever you are

Reply:
Same. Let's remove them whenever we come across them. No hurry.